components/navigation.js

```diff
@@ -507,7 +507,13 @@ function NavigationCollapsed({ navigationClick, toggleNavCollapsed }) {
         isNarrowNav
       >
         <NavigationIcon isNarrowNav>
-          <Icon.Brand />
+          <Image
+            src="/favicon.ico"
+            alt="clickable image"
```

Comment:
The `alt` tag should describe the element for visually impaired people, so I would say something like, “RegulomeDB logo that goes to the home page when clicked.”

Reply:
Thanks! Changed it!